packages/smart-contracts/scripts/2_deploy-main-payments.ts

```diff
@@ -105,7 +105,13 @@ export default async function deploy(args: any, hre: HardhatRuntimeEnvironment)
       ERC20Alpha:               ${erc20AlphaInstance.address}
       FakeSwapRouter:           ${FakeSwapRouterAddress}
       SwapToPay:                ${ERC20SwapToPayAddress}
-      `);
+    `);
+    return { 
+        DAI_address: erc20AlphaInstance.address, 
+        ERC20FeeProxy_address: ERC20FeeProxyAddress,
+        ERC20Test_address: testERC20Instance.address,
+        ETHFeeProxy_address: EthereumFeeProxyAddress
```

Comment:
Are we mixing camel case and snake case naming conventions here 😄

Reply:
Haha my bad, fixed